bundle/edu.gemini.pot/src/main/scala/edu/gemini/pot/sp/validator/SeqConstraint.scala

```diff
@@ -8,10 +8,15 @@ import scala.Some
 object SeqConstraint {
 
   // These types can occur in any sequence
-  val genericTypes = Types.initial
+  val genericTypes: Types = Types.initial
```

Comment:
:+1:  Explicit types make it much easier to read code.

Reply:
Agreed. I find myself so tired of surrounding a method with `{val xyz = .... }` to get the type.